src/rdf4cpp/rdf/datatypes/registry/DatatypeRegistry.hpp

```diff
@@ -273,6 +278,14 @@ class DatatypeRegistry {
      */
     [[nodiscard]] static InliningOps const *get_inlining_ops(DatatypeIDView datatype_id) noexcept;
 
+    inline static RuntimeConversionTable const *get_conversion_table(DatatypeIDView const datatype_id) noexcept {
```

Comment:
Is this used anywhere (except in get_supertype_conversion below)?

Reply:
I'm not sure what I originally used that for but it appears unused now.

Reply:
But I guess its a maybe useful accessor if you only want the table. If every other capability gets it's own accessor I guess this makes sense to have (?)